xdcs-agent/src/main/python/xdcs/servicers/AgentInfo.py

```diff
@@ -93,6 +95,8 @@ def _get_programs_from_path(self):
     def GetGPUInfo(self, request, context) -> GPUInfo:
         platforms = pyopencl.get_platforms()
 
+        pycuda.autoinit
```

Comment:
a to nie powinno być robione przy starcie aplikacji?

Reply:
tutaj wykryje zmiany w sprzecie w trakcie działania